.github/workflows/java-eks-e2e-test.yml

```diff
@@ -374,6 +374,7 @@ jobs:
           --service-name sample-application-${{ env.TESTING_ID }}
           --remote-service-name sample-remote-application-${{ env.TESTING_ID }}
           --remote-service-deployment-name ${{ env.REMOTE_SERVICE_DEPLOYMENT_NAME }}
+          --remote-resource-identifier "information_schema|${{env.RDS_MYSQL_CLUSTER_ENDPOINT}}|3306"
```

Comment:
Just confirming, have we double checked that this value is consistently this shape? What is the significance of `3306` here, is it always the same number?

Reply:
Port is always 3306 in the RDS cluster we created.